help/configure-emoticon-translations.md

```diff
@@ -1,7 +1,7 @@
-# Enable emoticon translation
+# Configure emoticon translation
```

Comment:
I added these changes to this commit, to fix the inconsistent singular and more importantly to make the old URL redirect to the new one, so as to not break links.
```
diff --git a/help/configure-emoticon-translations.md b/help/configure-emoticon-translations.md
index a4e7c72048..d1bacf5b3e 100644
--- a/help/configure-emoticon-translations.md
+++ b/help/configure-emoticon-translations.md
@@ -1,4 +1,4 @@
-# Configure emoticon translation
+# Configure emoticon translations
 
 If you use emoticons like `:)` or `:/`, you can configure whether to have them
 translated into emoji equivalents like
diff --git a/zerver/lib/url_redirects.py b/zerver/lib/url_redirects.py
index e9dcd37b17..b4b957bf76 100644
--- a/zerver/lib/url_redirects.py
+++ b/zerver/lib/url_redirects.py
@@ -72,6 +72,7 @@ HELP_DOCUMENTATION_REDIRECTS: List[URLRedirect] = [
     ),
     URLRedirect("/help/add-custom-emoji", "/help/custom-emoji"),
     URLRedirect("/help/night-mode", "/help/dark-theme"),
+    URLRedirect("/help/enable-emoticon-translations", "/help/configure-emoticon-translations"),
     URLRedirect("/help/web-public-streams", "/help/public-access-option"),
 ]
 
```